http/day.http

```diff
@@ -1,5 +1,5 @@
 ### create day
-POST {{host}}/days
+POST localhost:8080/api/v1/days
```

Comment:
You shouldn't change it, use {{host}} variable. You can switch environments this way.